src/views/v2/components/messages.php

```diff
@@ -20,16 +20,16 @@
 	return;
 }
 
-// @todo @juanfra Make this beautiful.
 ?>
 
-<div class="tribe-events-notices" style="text-align: center; width: 100%; margin: 0 0 1em auto;">
-	<?php
-	foreach ( $messages as $message_type => $message_group ) {
-		foreach ( $message_group as $message ) {
-			echo wp_kses_post( "{$message_type}: $message" );
-		}
-	}
-	?>
-</div>
-
+<?php foreach ( $messages as $message_type => $message_group ) : ?>
```

Comment:
I feel we should have a wrapper div for all these messages, so we can control all the messages with one div, what do you think?